docs/user-guide/index.md

```diff
@@ -57,6 +57,9 @@ the different pieces:
 * [Connectors](#connectors)
 * [Formatters](#formatters)
 
+## Permission Issue
```

Comment:
@qzhou1607 If there are no problems when converting it to HTML, I'm ok with that.

@sonarwhal/contributors Other opinions?